src/pages/Home/components/CriptoCard/CriptoCard.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {View} from 'react-native';
+import ImageComponent from 'src/components/Images/ImageComponent';
+import {SpinnerLoader} from 'src/components/Spinner/SpinnerLoader';
+import {useCriptoCard} from 'src/pages/Home/hooks/useCriptoCard';
+import {TextComponent} from 'src/styled-components/globalStyles/GlobalStyles.styled';
+import {defaultTheme} from 'src/styled-components/theme/theme.styled';
+import {
+  Cripto,
+  WrapNameAndIconCripto,
+  WrapPriceAndPercentCripto,
+  WrapStatusPercent,
+} from './styled-components/CriptoCard.styled';
+
+interface CriptoC {
```

Comment:
When you're defining an interface to use for props, you have to use a name like `ComponentNameProps`, in this case for example `CriptoCardProps`.

Reply:
Also, change Cripto to Crypto